src/preferences/dialog/dlgprefeq.cpp

```diff
@@ -672,11 +672,11 @@ void DlgPrefEQ::setUpMasterEQ() {
         comboBoxMasterEq->addItem(pManifest->name(), QVariant(pManifest->id()));
     }
     //: Displayed when no effect is selected
-    comboBoxMasterEq->addItem(tr("None"), QVariant());
+    comboBoxMasterEq->addItem(EffectsManager::kNoEffectString); // " --- "
```

Comment:
Duplicating the actual value of constants in comments is pointless. We don't want to repeat this string literal, not even in comments which would become stale and inconsistent if the value is updated

Reply:
I removed all duplicates in comments.